### PR TITLE
Implemented "build download" command

### PR DESCRIPTION
**Pull request by bryukov**
Implemented "build download" command.
capture

This PR depends on https://github.com/Microsoft/mobile-center-cli/pull/129

### CODE REVIEW

File: src/commands/build/category.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+TODO: Build command category description
```
Comment (bryukov) [inline src/commands/build/category.txt]: @lukekim,@mhr3, @siminapasat Could you, please, clarify what category description is expected for "build" command category?

**Comment by bryukov** [comment]
@christav Sorry, the PR doesn't need to include this changeset (it is essentially the same as in #129), I have removed it.

File: src/commands/build/category.txt (hunk 1 above)
Comment (siminapasat) [inline src/commands/build/category.txt]: @bryukov @lukekim Here's my proposal 
* mobile-center build * Commands for queueing builds, getting build status and downloading binaries and logs for completed builds
* mobile-center build queue* Queue a new build
* mobile-center build branches * Get information about branches and their latest build status
* mobile-center build download * Download the binary, logs or symbols for a completed build

Comment (bryukov) [inline src/commands/build/category.txt]: @siminapasat Thanks! I have updated the help messages.

**Pull request merged**